tests/e2e/e2e_port_exclusion_test.go

```diff
@@ -102,8 +102,8 @@ func testGlobalPortExclusion() {
 }
 
 func testPodLevelPortExclusion() {
-	const sourceName = "client"
-	const destName = "server"
+	const sourceName = "client1"
+	const destName = "server1"
```

Comment:
Please drop a comment here:
`// XXX(<gh-issue>): use a different namespace due to test pollution`

Reply:
@nshankar13 I suggested using `XXX` in the comments as an identifier for a hack.